ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -122,11 +145,18 @@ class ManagementResources(processCounter: ProcessCounter,
   private implicit final val plainBytes: FromEntityUnmarshaller[Array[Byte]] = Unmarshaller.byteArrayUnmarshaller
   private implicit final val plainString: FromEntityUnmarshaller[String] = Unmarshaller.stringUnmarshaller
 
-  private def withComment: Directive1[Option[String]] =
+  //if all deployment comments parsed come through here isn't it enough to validate that method?
+  private def withDeploymentComment: Directive1[Option[DeploymentComment]] = {
     entity(as[Option[String]]).map(_.filterNot(_.isEmpty)).flatMap {
```

Comment:
I don't like it. But right now I don't have idea how to change it..

Reply:
ok, i understand